static/admin/css/autocomplete.css

```diff
@@ -0,0 +1,260 @@
+select.admin-autocomplete {
```

Comment:
Remove these files. I think we do not need them. 
We can run collectstatic for these files. Put them in gitignore.